src/managers/PointerManager.cpp

```diff
@@ -635,7 +635,13 @@ Vector2D CPointerManager::closestValid(const Vector2D& pos) {
         float    distanceSq = __FLT_MAX__;
 
         for (auto& b : currentMonitorLayout.monitorBoxes) {
-            auto p      = b.closestPoint(vec);
+            auto p = b.closestPoint(vec);
+
+            auto maxPoint = Vector2D{std::nextafter(b.x + b.w, -INFINITY), std::nextafter(b.y + b.h, -INFINITY)};
```

Comment:
doesnt that in this case mean that maxPoint on a 1080p will be 1919,1079 which runs into problem numba 1 of 1919.5?

Reply:
no, on a 1080p `maxPoint` will be (1919.9999999999997726..., 1079.9999999999997726...)